lib/services/oauth2_service.ex

```diff
@@ -50,7 +50,9 @@ defmodule EaRestaurantDataLoader.Oauth2Service do
   end
 
   defp validate_client_credentials(client, client_secret, entity_status) do
-    case Bcrypt.verify_pass(client_secret, client.client_secret) do
+    {status, client_secret_decoded} = Base.decode64(client.client_secret, case: :mixed)
```

Comment:
Is this really validating the client secret?

Reply:
Yes. In a simple way it is comparing the decoded client secret from DB with client secret provided by the user.

Reply:
Done